Opserver.Core/Settings/RedisSettings.cs

```diff
@@ -50,9 +50,15 @@ public class Instance : ISettingsCollectionItem
             public string Password { get; set; }
 
             /// <summary>
+            /// Specify if ssl should be used. Defaults to false.
+            /// </summary>
+            public bool Ssl { get; set; } = false;
```

Comment:
You'll laugh but it was called that initially. Renamed just before the pull request.